apps/saleor-app-checkout/config/apl.ts

```diff
@@ -0,0 +1,23 @@
+import { FileAPL } from "@saleor/app-sdk/APL";
+import invariant from "ts-invariant";
+
+const apl = new FileAPL();
+
+export const get = async (saleorApiHost: string) => {
+  const authData = await apl.get(saleorApiHost);
+
+  invariant(authData, `No auth data found for given host: ${saleorApiHost}. Is the app installed?`);
+
+  const apiUrl = `https://${authData.domain}/graphql/`;
+
+  return {
+    apiUrl,
+    saleorApiHost,
+    domain: authData.domain,
+    appToken: authData.token,
+  };
+};
+
+export const set = ({ saleorApiHost, authToken }: { saleorApiHost: string; authToken: string }) => {
```

Comment:
I've found 3 names for seemingly the same thing: `token`, `authToken` and `appToken`. Are they the same?